net/netfilter/nf_tables_api.c

```diff
@@ -954,6 +954,25 @@ static void nf_tables_table_disable(struct net *net, struct nft_table *table)
 #define __NFT_TABLE_F_UPDATE		(__NFT_TABLE_F_WAS_DORMANT | \
 					 __NFT_TABLE_F_WAS_AWAKEN)
 
+static bool nft_table_pending_update(const struct nft_ctx *ctx)
+{
+	struct nftables_pernet *nft_net = nft_pernet(ctx->net);
+	struct nft_trans *trans;
+
+	if (ctx->table->flags & __NFT_TABLE_F_UPDATE)
+		return true;
+
+	list_for_each_entry(trans, &nft_net->commit_list, list) {
```

Comment:
```
net/netfilter/nf_tables_api.c:965:37: error: 'struct nftables_pernet' has no member named 'commit_list'
  list_for_each_entry(trans, &nft_net->commit_list, list) {
                                     ^~
```
`list_for_each_entry(trans, &ctx->net->nft.commit_list, list) {`
https://github.com/ctrliq/kernel-src-tree/blob/resf_kernel-4.18.0-553.16.1.el8_10/net/netfilter/nf_tables_api.c#L958-L975

Reply:
Fixed in latest - workflows all completed successfully.